ckanext/datastore/controller.py

```diff
@@ -15,24 +15,20 @@
     c,
     h,
 )
-from ckanext.datastore.writer import (
-    csv_writer,
-    tsv_writer,
-    json_writer,
-    xml_writer,
-)
+from ckanext.datastore.writer import get_writers
 from ckan.logic import (
     tuplize_dict,
     parse_params,
 )
 import ckan.lib.navl.dictization_functions as dict_fns
-
+from ckan.common import config
 from itertools import izip_longest
 
 int_validator = get_validator('int_validator')
 boolean_validator = get_validator('boolean_validator')
 
-DUMP_FORMATS = 'csv', 'tsv', 'json', 'xml'
+DUMP_FORMATS = config.get('ckan.dump_formats')
```

Comment:
@tino097 If `ckan.dump_formats` setting is not set in the `.ini` file, the `DUMP_FORMATS` variable will be `None`.

Reply:
Also, if it's set like this `ckan.dump_formats = csv tsv json xml` it should be parsed to be stored as an array.